### PR TITLE
Promo angular sb

### LINKED ISSUE #1858: Add Promo Component - Angular NDS

### PR DESCRIPTION
**Pull request by RVMendoza**
## What does this PR do?
Adding angular promos to storybook
  
promo

Note: flag looks looks bad in IE, and images are squished
Existing story for this -- https://github.com/sparkdesignsystem/spark-design-system/issues/1682

### Associated Issue 
Fixes #1858 
### Code
 - [x] Build Component in Spark Angular
 - [x] Unit Testing in Spark Angular with `gulp test-angular` (100% coverage, 100% passing)

### Browser Testing (current version and 1 prior)
  - [x] Google Chrome
  - [x] Google Chrome (Mobile)
  - [x] Mozilla Firefox
  - [x] Mozilla Firefox (Mobile)
  - [x] Microsoft Internet Explorer 11 (only this specific version of IE)
  - [x] Microsoft Edge
  - [x] Apple Safari
  - [x] Apple Safari (Mobile)

### CODE REVIEW

File: angular/projects/spark-angular/src/lib/components/sprk-promo/sprk-promo.stories.ts [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+      idString="promo-3"
+    >
+      Lorem ipsum dolor. Sit amet pede. Tempus donec et. Suspendisse id
+      inventore integer eum non enim diam habitant. Maecenas nunc per lacus
+      neque egestas. Diam quod curabitur.
+    </sprk-promo>
+  `,
+});
+
+
```
Comment (guern1kn) [inline angular/projects/spark-angular/src/lib/components/sprk-promo/sprk-promo.stories.ts]: Extra line here.

File: angular/projects/spark-angular/src/lib/components/sprk-promo/sprk-promo.stories.ts [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+export const withReversedImaged = () => ({
```
Comment (guern1kn) [inline angular/projects/spark-angular/src/lib/components/sprk-promo/sprk-promo.stories.ts]: Extra d in this name.

File: angular/projects/spark-angular/src/lib/components/sprk-promo/sprk-promo.stories.ts [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+      ctaText="Learn More"
+      idString="promo-4"
+    >
+      Lorem ipsum dolor. Sit amet pede. Tempus donec et. Suspendisse id
+      inventore integer eum non enim diam habitant. Maecenas nunc per lacus
+      neque egestas. Diam quod curabitur.
+    </sprk-promo>
+  `,
+});
+
```
Comment (guern1kn) [inline angular/projects/spark-angular/src/lib/components/sprk-promo/sprk-promo.stories.ts]: Few extra lines at the end of the file.

**Comment by afebbraro** [comment]
Soon as @guern1kn 's comments are fixed we can merge. Woot!

**Pull request merged**